src/sparseml/exporters/transforms/delete_repeated_qdq.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from onnx import ModelProto
+
+from sparseml.exporters.transforms.onnx_transform import OnnxTransform
+from sparseml.exporters.transforms.utils import assert_node_type
+from sparseml.onnx.utils import ONNXGraph
+
+
+__all__ = ["DeleteRepeatedQdq"]
+
+
+class DeleteRepeatedQdq(OnnxTransform):
+    """
```

Comment:
```suggestion
    """
    Removes a QDQ that immediately follows another QDQ.
    NOTE: this will modify graph outputs and does not
    guarantee identical graph behavior. This should be avoided
    and handled with better graph construction.  
```